contracts/test/integration/IntegrationTestPriceBoundPSMUSDC.t.sol

```diff
@@ -111,8 +111,9 @@ contract IntegrationTestPriceBoundPSMUSDCTest is DSTest {
     }
 
     /// @notice PSM is set up correctly and redeem view function is working
-    function testGetRedeemAmountOut() public {
-        uint256 amountVoltIn = 100e18;
+    function testGetRedeemAmountOut(uint256 amountVoltIn) public {
+        if (amountVoltIn > voltMintAmount) return;
+
```

Comment:
this test needs to test both the new PSM and the old PSM against each other to ensure the values are being calculated the same, which I am not seeing happening here.